test/integration/actions/action-create-card.spec.ts

```diff
@@ -643,6 +643,89 @@ describe('action-create-card', () => {
 		).rejects.toThrow(autumndbErrors.JellyfishPermissionsError);
 	});
 
+	test('should replace the file payload in data when payload exists in arguments', async () => {
+		const data = {
+			payload: {
+				file: {},
+			},
+			user: 'flowdock-user',
```

Comment:
Shouldn't make any direct references to things defined by plugins that are built on top of the worker, `flowdock` for example.